benchexec/containerexecutor.py

```diff
@@ -754,6 +754,13 @@ def child():
                         traceback.extract_tb(e.__traceback__, limit=-1)[0].line,
                         e,
                     )
+                    if util.try_read_file(
+                        "/proc/sys/kernel/apparmor_restrict_unprivileged_userns"
+                    ) == "1" and e.errno in [
+                        errno.EPERM,
+                        errno.EACCES,
+                    ]:
+                        logging.critical(container._ERROR_MSG_USER_NS_RESTRICTION)
```

Comment:
We could leave a comment with a link to https://ubuntu.com/blog/ubuntu-23-10-restricted-unprivileged-user-namespaces here.

Reply:
Should we replace this link with the corresponding blog for 24.04, https://ubuntu.com/blog/whats-new-in-security-for-ubuntu-24-04-lts , since this restriction became the default in 24.04?

Reply:
Hm, the newer blog post doesn't describe much, and it does not even mention the `sysctl` flag. And they do not even link to more information. So I guess the best link that we have remains the 23.10 blog post. Do you agree?

Reply:
Sure I agree with that 😃